client/components/domains/transfer-domain-step/index.jsx

```diff
@@ -380,11 +380,6 @@ class TransferDomainStep extends React.Component {
 
 		return (
 			<div className={ 'transfer-domain-step__domain-availability' }>
-				<Notice status="is-success" showDismiss={ false }>
```

Comment:
Do we need the importing of this component too? Assuming it isn't being used elsewhere:

https://github.com/Automattic/wp-calypso/blob/199dbcce32577681ef7485c7236be1391632e35a/client/components/domains/transfer-domain-step/index.jsx#L32

Reply:
Yep, it's used for the notice that shows up when you try to use an illegal character in the domain field.

Reply:
Yeah, sorry, just noticed it now too. I got disconnected from proxy and some error message showed up using the component.